Abha_Rana/Tic-Tac-Toe/tt.css

```diff
@@ -0,0 +1,32 @@
+* {
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+  }
+  .container {
+    display: flex;
+    background: #fff;
+    align-content: center;
+    justify-content: center;
+    flex-direction: column;
+    min-width: 700px;
+    min-height: 500px;
+    border-radius: 5%;
+    margin: auto;
+    position: relative;
+
+}
+  .heading-wrapper{
+    position: absolute;
+      top:0;
+      width:100%;
+  }  
+  .heading{
+      width:100%;
+      text-align: center; 
+     }
+  
+  table {
```

Comment:
never apply styling to an element in the main CSS file
start implementing the power of specificity

Reply:
elements are created in the script and hence do have any id or class. hence I had to apply styling to only those elements created using javascript.